test/testsets/GlassCat.jl

```diff
@@ -117,69 +113,182 @@ using StaticArrays
+            "     D₂:                                           0.0",
+            "     E₀:                                           0.0",
+            "     E₁:                                           0.0",
+            "     λₜₖ:                                          0.0",
+            "TCE (÷1e-6):                                       700.0",
+            "Ignore thermal expansion:                          false",
+            "Density (p):                                       0.878g/m³",
+            "ΔPgF:                                              0.008",
+            "RI at sodium D-Line (587nm):                       1.457587",
+            "Abbe Number:                                       57.19833",
+            "Cost relative to N_BK7:                            ?",
+            "Status:                                            Standard (0)",
+            "Melt frequency:                                    ?",
+            "Exclude substitution:                              false",
+            "Transmission data:",
+            "     Wavelength   Transmission      Thickness",
+            "         0.32μm           0.15         10.0mm",
+            "        0.365μm           0.12        100.0mm",
+            "       0.4047μm           0.42        100.0mm",
+            "         0.48μm           0.78        100.0mm",
+            "       0.4861μm           0.79        100.0mm",
+            "       0.5461μm           0.86        100.0mm",
+            "       0.5893μm            0.9        100.0mm",
+            "       0.6328μm           0.92        100.0mm",
+            "       0.6439μm            0.9        100.0mm",
+            "       0.6563μm           0.92        100.0mm",
+            "       0.6943μm           0.98        100.0mm",
+            "         0.84μm           0.99        100.0mm",
+            "      0.10648μm           0.61        100.0mm",
+            "         0.13μm           0.39        100.0mm",
+            "        0.155μm           0.11        100.0mm",
+            ""
+        ]
+        io = IOBuffer()
+        info(io, CARGILLE.OG0607)
+        @test String(take!(io)) === join(info_lines, '\n')
+
+        # TODO the rest of the string tests
+    end
+
+    @testset "search.jl" begin
+        @test glasscatalogs() == [
+            CARGILLE,
+            HOYA,
+            NIKON,
+            OHARA,
+            SCHOTT,
+            Sumita,
+        ]
+
+        @test glassnames(CARGILLE) == [
+            :OG0607,
+            :OG0608,
+            :OG081160,
+        ]
+
+        # TODO @test glassnames()
+
+        # !! TODO !! for some reason uncommenting this test causes the alloc tests at L145-146 to fail
+        # @test findglass(x -> (x.Nd > 2.1 && x.λmin < 0.5 && x.λmax > 0.9)) == [
+        #     HOYA.E_FDS3,
+        #     Sumita.K_PSFn214P,
+        #     Sumita.K_PSFn214P_M_,
+        # ]
+
+        # TODO _child_modules() unit test
```

Comment:
And this?

Reply:
probably not worth bothering with this at the moment as long as the functions which use this are ok